Explorer/Assets/Scripts/CrdtEcsBridge/Components/Defaults/PBMeshRendererDefaults.cs

```diff
@@ -0,0 +1,13 @@
+using DCL.ECSComponents;
+
+namespace CrdtEcsBridge.Components.Defaults
+{
+    public static class PBMeshRendererDefaults
```

Comment:
Let's agree on where we place defaults. I did it in the "feature folder" for another feature as they refer to the specific area. I propose to move them, WHYT?